### DISCUSSION TITLE
Unable to login to web calendar after successful install

### DISCUSSION THREAD

**Discussion by AlbaVore** [craigk5n/webcalendar#287] [Q&A]
As the title states I have installed webcalendar and cannot login in.
I have tried the default username and passwords as well as the users which exist in the database user list.

The installation went smoothly enough and i Have repeated the process a couple of times in order to check I was doin it correctly. But still I cannot login using any of the usernames and passwords I have in the database tables.

the link is here http://thegorbals.co.uk/thegorbalscalendar/

I should add that I was upgrading an existing install from 1.3 to 1.9. There was no problems during the upgrade until I try to login to the calendar.

Helpful insights are most welcome.

**Comment by scopemonkey** [comment]
Just to add to the original post, I am having the same problem. I upgraded from 1.3 to 1.9.1 and used the automatic installation script. The upgrade went fine except that none of my username and password combinations now work. I went into the SQL tables and deleted the default administrator user, then did the upgrade again enabling the default admin account. I confirmed this was recreated in the SQL tables. I was then able to login (but just once) using default admin/admin username/password combination. I created a totally new user set as an admin, logged out and tried to log back in with the new user credentials. It would not accept the user credentials just created. Also, it would no longer accept admin/admin that was just used. So, even if I create the default admin, it will only log in once and no other users can log in. That's great security if I forget to disable the default admin account, but kind of a problem for everyone else. Using PHP 7.4.29; File uploads ON; Allow URL open ON; Safe Mode OFF; GD ON; Database connections SUCCESSFUL; User Authentication is Web-Based via WebCalendar (default); Read Only NO; Environment PRODUCTION

**Comment by AlbaVore** [comment]
Hello scopemonkey, Did you manage to resolve this at all?
I had a solution that again worked once but now doesn't. I have tried a fresh install and deleting the old database and starting from scratch but still getting the same failed login with credentials I know are correct.
Let me know how you got on with this.

**Reply by scopemonkey** [reply]
I didn’t. I reverted back to v1.3 and stayed there. I never could find a way to get 1.9 to work. I haven’t revisited the issue since.

**Comment by AlbaVore** [comment]
hello again,

I tried the following:


Just to say that in phpmyadmin I changed the password type to varchar (255) and then with MD5 in the function tab for the admin user. This worked and allowed me to login. A real relief.

Let me know if it works for you.

Best wishes.

T: 0141 429 6514
M: 07740 856514
www.adevlinvisual.com

**Reply by scopemonkey** [reply]
Well, now I will have to revisit the issue and give that a try. Thanks. I'll let you know how it goes once I have time for the upgrade.
